enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/PatternExpressionImplementationAcceptanceTest.scala

```diff
@@ -337,8 +337,8 @@ class PatternExpressionImplementationAcceptanceTest extends ExecutionEngineFunSu
 
     executeWith(Configs.Interpreted, "MATCH (n:A) RETURN (n)-[:HAS]->() as p",
       planComparisonStrategy = ComparePlansWithAssertion((planDescription) => {
-        planDescription.find("Argument") shouldNot be(empty)
-        planDescription.cd("Argument").arguments should equal(List(EstimatedRows(1)))
+        planDescription.find("SingleRow") shouldNot be(empty)
```

Comment:
How can we test for SingleRow when plans never print this in the plan description?